gamestonk_terminal/stocks/stocks_controller.py

```diff
@@ -41,7 +28,7 @@
 )
 from gamestonk_terminal.common.quantitative_analysis import qa_view
 
-# pylint: disable=R1710
+# pylint: disable=R1710,import-outside-toplevel
```

Comment:
Should we add the import outside toplevel and too many lines into the pylintrc to ignore for all?

Reply:
too many lines I would add to pylintrc, the import outside I wouldn't because we only want this on the controllers and not on the rest.

One disadvantage of this import outside as well is that when something is wrong inside a menu, the terminal will only crash when we go into that same menu. 

We can probably add tests for this to try to go into each of the menus individually